OmniKit/Model/FaultEventCode.swift

```diff
@@ -120,194 +175,304 @@ public struct FaultEventCode: CustomStringConvertible {
         if let faultType = faultType {
             switch faultType {
             case .noFaults:
-                return LocalizedString("Fault Event Type 0x00: No faults, system running normally.", comment: "Fault Event Type 0x00: No faults, system running normally.")
+                return LocalizedString("Fault Event Type 0x00: No faults, system running normally", comment: "Fault Event Type 0x00: No faults, system running normally")
             case .failedFlashErase:
-                return LocalizedString("Fault Event Type 0x01: Flash erase failed in $4x00 page.", comment: "Fault Event Type 0x01: Flash erase failed in $4x00 page.")
+                return LocalizedString("Fault Event Type 0x01: Flash erase failed in $4x00 page", comment: "Fault Event Type 0x01: Flash erase failed in $4x00 page")
```

Comment:
I'm not sure we want the Localize these strings.  Currently they don't surface in Loop, and I'm not sure they should.  Only if the user is going to do different things based on the error.  I think currently all these are debugging/diagnostic level.  I.e. they should be reported, and maybe included in issue reports, but at the top level, I think we tell the user "Pod internal fault xx.  Please generate issue report, deactivate and replace your pod".

Reply:
The issue reports will be the best place to use them eventually for sure, I will dive into the message types to figure out how to do this properly. Nice thing to look into next week.